.vscode/extensions.json

```diff
@@ -2,6 +2,7 @@
   "recommendations": [
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
-    "bradlc.vscode-tailwindcss"
+    "bradlc.vscode-tailwindcss",
+    "redhat.vscode-yaml"
```

Comment:
Why's this extension needed?

Reply:
It gives auto-complete for GitHub's YAML syntax, and intellisense around documentation, from YAML Schema store.